chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2032,61 +2034,16 @@ private boolean tryLoadDump(RenderContext context, String fileName, TaskTracker
       }
       return false;
     }
-    try(PushbackInputStream input = new PushbackInputStream(new FileInputStream(dumpFile), 4)) {
-      byte[] magicNumber = new byte[4];
-      input.read(magicNumber, 0, 4);
-      if(Arrays.equals(magicNumber, DUMP_FORMAT_MAGIC_NUMBER)) {
```

Comment:
This logic must not be removed. This PR breaks compatibility with the old dump format.

Reply:
This logic for handling the compression and format version logic can be found in `RenderDump`. :)